ofctrl/fgraphFlow.go

```diff
@@ -557,6 +559,16 @@ func (self *Flow) xlateMatch() openflow13.Match {
 		ofMatch.AddField(*ndTllField)
 	}
 
+	// Handle pkt_mark match
+	if self.Match.PktMark != 0 {
```

Comment:
Just would like to confirm that PktMark could be 0 and we just don't need 0 to be matched, right?

Reply:
Good question, I think there may be a requirement to match pkt_mark=0 in future, then we need to write something like PktMark=0, PktMarkMask=0xffffffff but it wouldn't work. 
However, all int type fields in this file are currently written in this way, for example, self.Match.CtMark. Maybe they should be changed together later when the requirement becomes true.

Reply:
Got it, thanks.